# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1360603447 18000
#      Mon Feb 11 12:24:07 2013 -0500
# Node ID 5919794d640a53aa6f360d4d2572b3dfb7fb4b3b
# Parent  dd583c258cf97125780c5757e4d20584db78af65
# Parent  bbef6907d766321e54ce8ddba01bd195e9b4b149
maint: periodic merge of stable to default

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-fea72d94801364b1bde7cdf83237a49fe4db508a gnulib-hg
+0eef0a2a05e22ac8ef2aad06dda200f7def83396 gnulib-hg
